hw2/02-reverse.html

```diff
@@ -23,7 +23,9 @@ <h1 class="mt-2 mb-4">Reverse</h1>
           class="btn btn-primary"
           value="Reverse"
           id="reverse"
+          onclick="reverse(parseInt(document.querySelector('#input').value))"
```

Comment:
This is a really good idea. I wish I would have realized that I could have just manipulated the html file for the on click event. I spent a lot of time trying to figure it out in JS.